pkg/manifest/manifest.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/cbroglie/mustache"
 	"github.com/cnabio/cnab-go/bundle"
 	"github.com/cnabio/cnab-go/bundle/definition"
+	"github.com/dustin/go-humanize"
```

Comment:
Nit: This package seems to be not very maintained, but it doesn't seem to really need a lot. We've been using it already, so it's really nbd, but with recent security things going on I think I'm just hyper-aware for a bit :)
Just thinking out loud here.